index.html

```diff
@@ -875,6 +875,16 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
           Location Discovery</a>.
         </p>
       </section>
+      <section id="append-mode">
+        <h2>Append Mode</h2>
+        <p>
+          The [[!SOLIDWEBAC]] <a href="https://github.com/solid/web-access-control-spec#modes-of-access">Modes of
+          Access</a> section mentions the <code>acl:mode</code> value <code>acl:Append</code>.  In the context of a
+          Fedora implementation, <code>acl:Append</code> should be understood as operations that only append, such as
+          <code>POST</code>ing to a container, or performing a <code>PUT</code> or <code>PATCH</code> that only adds
```

Comment:
We should also consider PUT to creating resources as well as just POSTing.

Reply:
@dannylamb there is some disagreement about whether create-on-PUT adds `ldp:contains` triples to the parent resource. While Fedora4 allows this behavior, other implementations may not, and after much discussion, the Fedora specification has, thus far, decided to remain silent on this point. @escowles' text above (in my reading) encompasses the possibility that a PUT would add triples to the repository -- whether that relates to the target resource or some other resource such as a parent container.

Reply:
@acoburn @escowles Fair enough.

Reply:
I would be inclined to refine `acl:Append` to only include:
- `POST` to a container, and
- `PATCH` to an LDPRS that is only additive.

Although the effect of a PUT that replaces triples with the original plus additional triples is additive, the process is a full over-write.

Reply:
@awoods I agree it's harder to tell if a PUT is only additive or not.  I've removed it from the list of examples.

@dannylamb @zimeon My new commit has dismissed your reviews, so please take a look again.